### ISSUE TITLE
`export default` for AMD broken in 2.7.0

### ISSUE THREAD

**Issue by caseywebdev** [6to5/6to5#415]
By using `module.exports = foo` instead of `exports['default'] = foo`, any `export var bar = baz` statements are broken.

```js
export var bar = baz;
export default foo;
```

used to be

```js
var bar = exports.bar = baz;
exports['default'] = foo;
```

but is now (broken in 2.7.0)

```js
var bar = exports.bar = baz;
module.exports = foo;
```

`module.exports` essentially overwrites `exports` and then anything attached to `exports` is really disregarded.

**Comment by sebmck** [comment]
It's actually turned into

```javascript
var bar = exports.bar = baz;
exports["default"] = foo;
module.exports = _extends(exports["default"], exports);
```

Which will work.

**Comment by caseywebdev** [comment]
Right, it does that for CJS, but not AMD.

```node
> console.log(to5.transform('export var bar = baz;\nexport default foo;', {modules: 'amd'}).code)
define(["exports", "module"], function (exports, module) {
  "use strict";

  var bar = exports.bar = baz;
  module.exports = foo;
});
```

**Comment by sebmck** [comment]
Oh right, duh. Looks like this was broken by #379...

**Comment by sebmck** [comment]
Thanks! Fixed as of 2.8.0. I'll be sure to vet PRs more in the future, sorry!

**Comment by caseywebdev** [comment]
No worries! It's awesome you're so active with this project!
